packages/slate-tag-webpack-plugin/index.js

```diff
@@ -9,7 +10,11 @@ FileListPlugin.prototype.apply = function(compiler) {
     const asset = compilation.assets[SETTINGS_SCHEMA_PATH].source();
     const schema = JSON.parse(asset);
 
-    if (Array.isArray(schema) && typeof schema[0] === 'object') {
+    if (
+      Array.isArray(schema) &&
+      typeof schema[0] === 'object' &&
+      schema[0].name === THEME_INFO_PANEL
```

Comment:
Instead of needing it to be first settings panel, can we just find the schema item with `name == THEME_INFO_PANEL`?

Reply:
@t-kelly at first I thought `theme_info` had to be first but double-checked and looks like it can be anywhere. Will update the PR.

Reply:
Done.